contracts/StrategySushiswapPair.sol

```diff
@@ -84,7 +84,7 @@ contract StrategySushiswapPair is BaseStrategy {
      * Provide an accurate expected value for the return this strategy
      * would provide to the Vault if `report()` was called right now
      */
-    function expectedReturn() public override view returns (uint256 _liquidity) {
+    function expectedReturn() internal view returns (uint256 _liquidity) {
```

Comment:
looks like you are not calling this anymore. You can remove the method.

Reply:
I am actually still using it [here](https://github.com/andy8052/strategy-sushi/pull/1/files#diff-31bca2dbef069066357afbbb8484cf779d5d14fbe57a0a63ab98e0dd4aeefc9bL120)